examples/videocomposer/vc/config/parser.py

```diff
@@ -75,6 +75,7 @@ def _parse_args(self):
         )
         parser.add_argument(
             "--style_image",
+            default="",
```

Comment:
After setting `style_image` to an empty string, will `read_image_if_provided` fail because it can't open an empty path with `img = Image.open(open(path, mode="rb")).convert("RGB")`?

Reply:
should set None as default, so that it can be overwritten by yaml config arg.